.classpath

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Don't include .classpath in your pull request. Download the gitignore from master

Reply:
Sure, I'll delete those files